### PR TITLE
fix(APP-3805): Fix color of body name on ProposalVoting.BodyContext module component

### PR DESCRIPTION
**Pull request by shan8851**
## Description

Add correct text color to ProposalVotingBodyContent component

Task: [APP-3805](https://aragonassociation.atlassian.net/browse/APP-3805)

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] I have selected the correct base branch.
- [x] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [ ] Any dependent changes have been merged and published in downstream modules.
- [x] I ran all tests with success and extended them if necessary.
- [x] I have updated the `CHANGELOG.md` file in the root folder of the package after the [UPCOMING] title and before the
      latest version.
- [ ] I have tested my code on the test network.

[APP-3805]: https://aragonassociation.atlassian.net/browse/APP-3805?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -18,6 +18,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 - Update minor and patch NPM dependencies
 - Support `type` property on the buttons of the `<Dialog.Footer />` core component
+- Update text color on `ProposalVotingBodyContent` module component
```
Comment (cgero-eth) [inline CHANGELOG.md:21]: I'd move this to the `fixed` section and change it to:
```
Fix color of body name on `<ProposalVoting.BodyContext />` module component
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -14,6 +14,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - Update `ProposalActions` module component to improve rendering of decoded parameters and support and "edit" mode
   through the `editMode` property
 
+### Fixed
```
Comment (cgero-eth) [inline CHANGELOG.md:17]: The new entry should be under the `unreleased` section, seeing this now sorry 😬

Comment (shan8851) [reply CHANGELOG.md]: My bad, I missed that, as I think it changed between when I started working on it and now. Updated

**Review by cgero-eth — APPROVED** [review]
LGTM thanks @shan8851!

**Pull request merged**